Description: Workaround for bundler
 bundler can't ignore groups when --local is specified for bundle install
Bug: https://github.com/bundler/bundler/issues/4023

--- a/Gemfile
+++ b/Gemfile
@@ -224,7 +224,8 @@ group :production do # we don"t install these on travis to speed up test runs
   gem "asset_sync", "~> 2.5", require: false
 end
 
-group :development do
+#group :development do
+if ENV["INCLUDE_TEST_DEPENDS"] == "true"
   # Automatic test runs
   gem "guard",          "2.15.0", require: false
   gem "guard-rspec",    "4.7.3", require: false
@@ -255,9 +256,9 @@ group :development do
   gem "simplecov", "0.16.1", require: false
 
   gem "turbo_dev_assets", "0.0.2"
-end
+#end
 
-group :test do
+#group :test do
   # RSpec (unit tests, some integration tests)
 
   gem "fixture_builder",   "0.5.2.rc3"
@@ -284,9 +285,9 @@ group :test do
 
   # Coverage
   gem "coveralls", "0.8.22", require: false
-end
+#end
 
-group :development, :test do
+#group :development, :test do
   # RSpec (unit tests, some integration tests)
   gem "rspec-rails", "3.8.1"

